### DISCUSSION TITLE
How does remarking by HOD work?

### DISCUSSION THREAD

**Discussion by Pushpavel** [Pushpavel/project-academics#3] [Q&A]
According to @Chandrasekar sir's specification, 
- HOD can remark a course if it's ```GRADES``` document is in ```submitted``` state
- remarking a course means all of its documents go to the ```remarked``` state
- The handling faculty has to resubmit every document and finally the ```GRADES``` document

#### Things needing attention
- when a course gets remarked, it seems very inefficient to both the faculty and backend resources
  - faculty needs to resubmit every academic document in the course even if only one needs attention
  - backend needs to perform copying of private document entries to protected document for each one of these resubmitted academic documents even though it doesn't have any change
- HOD's remark message would be intended for an academic document that needs correction. remarking every academic document and displaying this message on each one of them seems counter-intuitive

**Comment by moseschand003** [comment]
😂😂😂. What's your idea ?
